jetty-server/src/main/config/etc/jetty-bytebufferpool.xml

```diff
@@ -8,5 +8,7 @@
     <Arg type="int"><Property name="jetty.byteBufferPool.maxQueueLength" default="-1"/></Arg>
     <Arg type="long"><Property name="jetty.byteBufferPool.maxHeapMemory" default="0"/></Arg>
     <Arg type="long"><Property name="jetty.byteBufferPool.maxDirectMemory" default="0"/></Arg>
+    <Arg type="long"><Property name="jetty.byteBufferPool.retainedHeapMemory" default="0"/></Arg>
+    <Arg type="long"><Property name="jetty.byteBufferPool.retainedDirectMemory" default="0"/></Arg>
```

Comment:
@lorban With this simple/minimal change to the exist module/xml we get to configure both types of buffer pool in one.

Reply:
Which is both very nice and desirable. :+1: